unable to install evil

I get an error when trying to install evil.rcp under el-get. Cloning the repo and doing a "make all" seems to work fine on its own.

```
Compute dependencies
emacs --batch -Q -L . -L lib -f batch-byte-compile evil-vars.el
Wrote /Users/mark/.emacs.d/el-get/evil/evil-vars.elc
emacs --batch -Q -L . -L lib -f batch-byte-compile evil-common.el

In toplevel form:
evil-common.el:29:19:Warning: reference to free variable `evil-visual-state-p'

In evil-line-move:
evil-common.el:1103:18:Warning: reference to free variable `line-move-visual'

In end of data:
evil-common.el:3205:1:Warning: the following functions are not known to be
    defined: declare-function, global-undo-tree-mode, characterp,
    beginning-of-visual-line, region-active-p, evil-visual-state-p,
    match-substitute-replacement
Wrote /Users/mark/.emacs.d/el-get/evil/evil-common.elc
emacs --batch -Q -L . -L lib -f batch-byte-compile evil-digraphs.el
Wrote /Users/mark/.emacs.d/el-get/evil/evil-digraphs.elc
emacs --batch -Q -L . -L lib -f batch-byte-compile evil-core.el

In toplevel form:
evil-core.el:101:1:Error: Symbol's value as variable is void: evil-visual-state-p
make: *** [evil-core.elc] Error 1
```

--- a/recipes/evil.rcp
+++ b/recipes/evil.rcp
@@ -9,4 +9,5 @@
        :depends undo-tree
        :build (("make" "all" "info"))
        :build/berkeley-unix (("gmake" "all" "info"))
+       :build/darwin `(("make" ,(format "EMACS=%s" el-get-emacs) "all" "info" ))
        :info "doc")

